lib/routes/api/accounts.js

```diff
@@ -16,11 +16,15 @@ const {hasAccountPermissions, parseAccountSid, enableSubspace, disableSubspace}
 const short = require('short-uuid');
 const VoipCarrier = require('../../models/voip-carrier');
 const translator = short();
+const SpeechCredential = require('../../models/speech-credential');
 
 let idx = 0;
 
 const getFsUrl = async(logger, retrieveSet, setName) => {
-  if (process.env.K8S) return `http://${process.env.K8S_FEATURE_SERVER_SERVICE_NAME}:3000/v1/createCall`;
+  if (process.env.K8S) {
+    const port = process.env.K8S_FEATURE_SERVER_SERVICE_PORT || 3000;
```

Comment:
is this a new env var?

Reply:
yes, it's new env var to allow me create fs_scaffold in different port locally. default value is 3000, so there is no breaking change if system does not provide this